src/lib/storage/vector_compression/simd_bp128/simd_bp128_decompressor.hpp

```diff
@@ -2,6 +2,7 @@
 
 #include <array>
 #include <memory>
+#include <mutex>
```

Comment:
@Bouncner why is that not necessary in the master branch?

Reply:
The master does not use a lock. There is an issue about potential MT problems, but we decided against a lock a couple of months ago.